ui/app/components/http-requests-table.js

```diff
@@ -0,0 +1,51 @@
+import Component from '@ember/component';
+import { computed } from '@ember/object';
+import { assign } from '@ember/polyfills';
+
+/**
+ * @module HttpRequestsTable
+ * HttpRequestsTable components render a table with the total number of HTTP Requests to a Vault server per month.
+ *
+ * @example
+ * ```js
+ * const COUNTERS = [
```

Comment:
I'm not sure about showing examples this way. It's nice to show the shape of the data, but also you could never actually use the components this way. What do you think about documenting the shape of the data in the `@param` below?

Reply:
oh, i like that _way_ better. great point. will update now, thank you!